cmd/cap-release/render.go

```
@@ -0,0 +1,154 @@
+}
+
+// atomicWrite writes content to a temp file in the same directory then renames it
+// over path so a partial write never corrupts the target.
+func atomicWrite(path, content string) error {
+	f, err := os.CreateTemp(filepath.Dir(path), ".cap-release-*")
+	if err != nil {
+		return err
+	}
+	tmp := f.Name()
+	if _, err := f.WriteString(content); err != nil {
+		f.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `f.Close` is not checked

(errcheck)